docs/content/preview/explore/change-data-capture/using-logical-replication/yugabytedb-connector.md

```diff
@@ -1095,6 +1095,10 @@ YugabyteDB connector events are designed to work with [Kafka log compaction](htt
 
 When a row is deleted, the _delete_ event value still works with log compaction, because Kafka can remove all earlier messages that have that same key. However, for Kafka to remove all messages that have that same key, the message value must be `null`. To make this possible, the YugabyteDB connector follows a _delete_ event with a special tombstone event that has the same key but a `null` value.
 
+## Updating or deleting a row which was inserted in the same transaction
```

Comment:
I think this should be documented in the gRPC connector section as well.

Reply:
It is present in the `cdc-get-started.md` for gRPC protocol.